server/models/user.js

```diff
@@ -24,12 +48,40 @@ module.exports = function(sequelize, DataTypes) {
           callback(res);
         });
       },
-      toJson: function() {
+      toJson: function(currentUserId) {
```

Comment:
@dwoznicka In my opinion all `toJson` methods should be independent of current user. The idea was that `toJson` method serialize database object to JSON string, and you are using it as a kind of specific utility function. I think you should create your own function which serialize object using 'toJson' and extends that serialized object with current user releated informations.